packages/discord.js/src/structures/DMChannel.js

```diff
@@ -81,7 +98,7 @@ class DMChannel extends Channel {
    * console.log(`Hello from ${channel}!`);
    */
   toString() {
-    return this.recipient.toString();
+    return Formatters.userMention(this.recipientId);
```

Comment:
Instead of using the Formatters namespace class, you should use builder's function directly.

Reply:
Should be done with 41097f3ae44d1d4ee43015ed75c6d6c125e1c3d1.